mitype/app.py

```diff
@@ -447,6 +455,28 @@ def print_stats(self, win):
             curses.color_pair(5),
         )
 
+    def switch_text(self, win, value):
+        """Load next or previous text snippet from database.
+
+        Args:
+            win (any): Curses window.
+            value (int): value to increase or decrement by.
+        """
+        if isinstance(self.text_id, str):
+            return None
```

Comment:
Can we just use `return` instead of `return None`? Ideally, using `return None` suggests that in the later part of the function, there's more to see and something more interesting is returned. Even though there's no functional difference, such small hints improve the readibility.